src/app/features/dialogs/high-fee-dialog/high-fee-dialog.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from 'react';
```

Comment:
I could do with some help testing this to make sure I haven't broken it. I went down a rabbit hole in [this commit](https://github.com/leather-wallet/extension/pull/4655/commits/aaee818ac674d12e52729bf275b69dc06ab4ebcc) trying to clean up the `jotai` state for this dialog. 

It was pretty hard to follow and I think we'd be better off just showing this dialog on demand when fees are too high rather than storing if it's shown or not in global state

Reply:
The link here to the commit is outdated. I thought the high fee warning was just controlled by the `useDrawer` hook and shared? What was handled by jotai?

Reply:
When I ref the current code it was just a shared hook that was triggered in the forms and set to true (to display) if the fee was detected as higher than the global shared constant we set. I don't think jotai was involved?

Reply:
O, nvm, I see it now. That was stored in global state. The handling of those drawers with jotai was not setup by me but I prob did add to it a long time ago. I can try to test if the changes here are working.

Reply:
Would it help at all to make this a slice of redux state to handle the ui?

Reply:
Thanks Fara! Yes we could try make a slice but I don't think it's a good pattern to store the open status of dialogs globally. 

I am handling it on a more local level now so I thin that should be OK. 

I have removed the `useDrawers` hooks

Reply:
We could also handle just with a route later, but not nec here.